app/src/main/kotlin/com/mapzen/erasermap/util/StringConstants.kt

```diff
@@ -0,0 +1,15 @@
+package com.mapzen.erasermap.util
+
+/**
+ *  Declaration for all string constants to be used in the app
+ */
+public class StringConstants {
```

Comment:
This is a matter of style but in general I'm not a big fan of an app-wide class for constants that don't necessarily relate to each other.

Seems like these values would be better served in the class to which they are relevant. In this case maybe we could add them to the `AppSettings` interface?